src/modules/homeView.js

```diff
@@ -0,0 +1,144 @@
+import { likeRocket, getLikes, getComments } from './apis.js';
+import { popupView } from './commentPopup.js';
+
+const homeContainer = document.querySelector('#home-container');
+
+// render data of the homepage
+const homepageView = async (data) => {
+  const rocketsContainer = document.querySelector('.rockets-container');
+  let rockets = '';
+
+  try {
+    const likesData = await getLikes(); // Retrieve the likes data from the API
+    const likesMap = new Map();
+
+    /* eslint-disable camelcase */
+    likesData.forEach(({ item_id, likes }) => {
+      // Create a map of rocketId to likes count
+      likesMap.set(item_id, likes);
+    });
+
+    data.forEach((rocket) => {
+      /* eslint-disable camelcase */
```

Comment:
- [ ] Please avoid disabling linter rules as they serve as a way to provide a consistent code format and ensure you use best practices. Instead, consider fixing the errors! 🤓 🔧

Reply:
Kindly check the api yourself and find out the variable naming conventions used there. I'm only making use of the data I can't change it. That is one of the things why eslint made "disable" commands, to be used when necessary

Reply:
resolved